sale_exception_product_sale_manufactured_for/models/sale_order.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+from odoo import models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def exception_partner_can_order(self):
```

Comment:
```suggestion
    def exception_partner_can_order_manufactured_for(self):
```
Just to avoid a too generic name, which could be conflicting in the future.

Reply:
Thanks, I have changed it in demo data too